robocop/checkers/naming.py

```diff
@@ -75,8 +75,9 @@ class KeywordNamingChecker(VisitorChecker):
     rules = {
         "0302": (
             "not-capitalized-keyword-name",
```

Comment:
'capitalized' is still in rule name, but changing it would be not backward compatible

Reply:
We can provide new rule instead of this one and this can be tagged as deprecated and then we can use whatever is more suitable. What do you think?

Reply:
I agree, and we can add print with deprecation warning only if someone includes / excludes deprecated rule (thanks for the PR with verification if rule exist if used in config it's now possible)

Reply:
I've replaced the names but now I wonder if it's too explicit. 'not-title-case-keyword-name' with convention param 'first_word_capitalized' would not actually check for title case but capitalization. So maybe more general `wrong-capitalization-in-keyword-name` or similiar?

Reply:
I think the name `wrong-case-in-keyword-name` for the parameter should work fine.